docs/WritingTests.asciidoc

```diff
@@ -528,20 +525,21 @@ sub run {
 ====
 
 
-Mutexes are a way to wait for specific event from single job.
-When we need multiple jobs to reach required state we need to use barriers.
-
-Before first use barrier needs to be created with barrier_create with 2 parameters - name and count.
-Name behaves as ID (same as with mutexes), count is number of jobs needed to call barrier_wait to unlock barrier.
+Mutexes are a way to wait for specific events from a single job.
+When we need multiple jobs to reach a certain state we need to use barriers.
 
-There is optional barrier_wait parameter called check_dead_job.
-When used it will kill all jobs waiting in barrier_wait if one of cluster jobs die.
+To create a barrier call +barrier_create+ with the parameter name and count.
+The name serves as an ID (same as with mutexes). The count parameter specifies the
+number of jobs needed to call +barrier_wait+ to unlock barrier.
 
-It prevents waiting for state that will never be reached (and eventually die on job timeout).
-Should be set only on one of barrier_wait calls.
+There is an optional +barrier_wait+ parameter called +check_dead_job+.
+When used it will kill all jobs waiting in +barrier_wait+ if one of the cluster jobs
+dies. It prevents waiting for states that will never be reached (and eventually dies
+on job timeout). It should be set only on one of +barrier_wait+ calls.
```

Comment:
Huh, you forgot to add an article. How could this have happened to *you* ;)

Reply:
Maybe I have been reading too much Russian English myself now :-)

Reply:
just for the record: the more (politically) correct term is english following slavic grammar - educate yourself: https://en.wikipedia.org/wiki/Grammatical_aspect_in_Slavic_languages (and note no russian was involved in writing the original)